### ISSUE TITLE
1.2.0 version output to kafka panic

### ISSUE THREAD

**Issue by dujiang426** [buger/goreplay#848]
I used the following command to be ok in version 0.16.1, but a panic error was reported in version 1.2.0.

`sudo ./gor --input-raw :8080 --output-kafka-host '192.168.0.1:9092,192.168.0.2:9092' --output-kafka-topic 'kafka-log'`

```
panic: reflect: Call with too many input arguments

goroutine 1 [running]:
reflect.Value.call(0x488e500, 0x4a15b60, 0x13, 0x49c1c1e, 0x4, 0xc000096f60, 0x3, 0x4, 0x1, 0x0, ...)
	/usr/local/go/src/reflect/value.go:380 +0x134c
reflect.Value.Call(0x488e500, 0x4a15b60, 0x13, 0xc000096f60, 0x3, 0x4, 0x2, 0x4, 0x1)
	/usr/local/go/src/reflect/value.go:321 +0xb4
main.(*InOutPlugins).registerPlugin(0xc0000e0780, 0x488e500, 0x4a15b60, 0xc00021fda8, 0x3, 0x3)
	/Users/leonidbugaev/go/src/github.com/buger/goreplay/plugins.go:50 +0x1f4
main.NewPlugins(0xc0001ac240)
	/Users/leonidbugaev/go/src/github.com/buger/goreplay/plugins.go:140 +0x52f
main.main()
	/Users/leonidbugaev/go/src/github.com/buger/goreplay/gor.go:53 +0x8bc
```

**Comment by urbanishimwe** [comment]
It was fixed by the PR linked to #835

**Comment by dujiang426** [comment]
> It was fixed by the PR linked to #835

ok
